SU-SWS/stanford_basic
# D8CORE-1286: Search page styles refactoring.

# READY FOR REVIEW 

# Summary
- Changes from our demo

# Review By (Date)
- End of the week

# Urgency
- Super low

# Steps to Test

1. Check out this branch
2. Compile
3. Validate that the styles are available on the search page
